.pre-commit-config.yaml

```diff
@@ -69,7 +69,7 @@ repos:
             "--load-plugins=pylint.extensions.docparams",
           ]
         # disabled plugins: pylint.extensions.mccabe
-        exclude: tests/functional/|tests/input|tests/extensions/data|tests/regrtest_data/|tests/data/|doc/
+        exclude: tests/functional/|tests/input|tests/regrtest_data/|tests(/\w*)*/data/|doc/
```

Comment:
That's great, maybe we could add it line 16 in the "fixtures" variable too ?

Reply:
I'll do a separate PR for the pre-commit changers

Reply:
See #5974.